lib/tokens/session_token.js

```diff
@@ -24,11 +24,7 @@ module.exports = (log, inherits, Token, config) => {
     this.locale = details.locale || null
     this.mustVerify = !! details.mustVerify || false
 
-    if (details.createdAt > 0) {
-      this.accountCreatedAt = details.createdAt
-    } else {
-      this.accountCreatedAt = null
-    }
+    this.accountCreatedAt = details.accountCreatedAt || null
```

Comment:
Previous discussion in https://github.com/mozilla/fxa-auth-server-private/pull/45/files#r95722719 for context